### DISCUSSION TITLE
How to sort files based on numbers?

### DISCUSSION THREAD

**Discussion by Timsword** [community/community#23969] [New to GitHub]
<p>Hi everyone.</p>
<p>I just created a repositorie and uploaded some files inside of it. in my pc, i was sorting them based on their numbers at front of their titles. But when i uploaded the files, github directly sorted them alphabetical, without any exception. So is there is a way to sort them based on their numbers?</p>
<p><img src="https://user-images.githubusercontent.com/12910594/181080444-dda07adc-3705-453b-8859-741b68648480.png" alt="image" data-base62-sha1="40bxFo0atYC61E12ppz0MG4hUsb" width="110" height="487"></p>

**Comment by kingthorin** [comment, marked as answer]
<p>Leading zeros (on the single digit numbers) should do it.</p>

**Reply by abubakar-sadiq001** [reply]
These is really helpful thank you sir.

**Reply by JinLeeGG** [reply]
Thank you so much
